sharding/src/main/resources/application.conf

```diff
@@ -96,5 +100,11 @@ shard-bench {
   pings-per-second = 800
   number-of-shards = 100
   total-nodes = 1
+  
+  
+  # actor counting scalability -------------
+  add-actors-batch = 100
+  add-actors-interval = 5.seconds
+  # end of actor counting scalability ------
```

Comment:
could it start next batch immediately after the previous is done (or with a short delay between)?